src/Functions/IFunction.cpp

```diff
@@ -191,7 +191,7 @@ ColumnPtr IExecutableFunction::defaultImplementationForNulls(
 
     NullPresence null_presence = getNullPresense(args);
 
-    if (null_presence.has_null_constant || null_presence.has_nullable)
+    if (null_presence.has_null_constant)
```

Comment:
The 2 if conditions are now the same

Reply:
Please don't ignore this comment

Reply:
This code is now doing:

```
if (a)
{
   // do X
}

if (a)
{
  // do Y
}
```

It should be:
```
if (a)
{
  // do X
  // do Y
}
```

Reply:
get..